libs/coin-tester/package.json

```diff
@@ -0,0 +1,78 @@
+{
+  "name": "@ledgerhq/coin-tester",
+  "version": "0.1.0",
+  "description": "Deterministic testing of Ledger coin-modules",
+  "license": "Apache-2.0",
+  "scripts": {
+    "clean": "rimraf lib lib-es",
+    "build": "tsc && tsc -m ES6 --outDir lib-es",
+    "prewatch": "pnpm build",
+    "watch": "tsc --watch",
+    "doc": "documentation readme src/** --section=API --pe ts --re ts --re d.ts",
+    "lint": "eslint ./src --no-error-on-unmatched-pattern --ext .ts,.tsx --cache",
+    "lint:fix": "pnpm lint --fix",
+    "test": "jest"
+  },
+  "dependencies": {
+    "@ledgerhq/coin-framework": "workspace:^",
```

Comment:
Those dependencies are not necessary: `coin-framework`, `cryptoassets`, `hw-app-eth`, `hw-transport`.
Could you check what is the minimum required (I see also bignumber and rxjs and not sure where they are used).